src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1137,6 +1136,36 @@ static bool _attrParseCssStyleNode(void* data, const char* key, const char* valu
 }
 
 
+static bool _attrParseSymbolNode(void* data, const char* key, const char* value)
+{
+    SvgLoaderData* loader = (SvgLoaderData*)data;
+    SvgNode* node = loader->svgParse->node;
+    SvgSymbolNode* symbol = &(node->node.symbol);
+
+    if (!strcmp(key, "viewBox")) {
+        if (_parseNumber(&value, &symbol->vx)) {
+            if (_parseNumber(&value, &symbol->vy)) {
+                if (_parseNumber(&value, &symbol->vw)) {
+                    _parseNumber(&value, &symbol->vh);
+                }
+            }
+        }
+    } else if (!strcmp(key, "width")) {
+        symbol->w = _toFloat(loader->svgParse, value, SvgParserLengthType::Horizontal);
+    } else if (!strcmp(key, "height")) {
+        symbol->h = _toFloat(loader->svgParse, value, SvgParserLengthType::Vertical);
+    } else if (!strcmp(key, "preserveAspectRatio")) {
+        if (!strcmp(value, "none")) symbol->preserveAspect = false;
+    } else if (!strcmp(key, "overflow")) {
+        if (!strcmp(value, "visible")) symbol->overflowVisible = true;
+    } else {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/overflow (`This feature is not widely implemented yet.`)
Should we support overflow property?
It seems that this property is not yet that should be supported officially(?).

Reply:
I checked [SVG2](https://www.w3.org/TR/SVG2/struct.html#SymbolNotes) 
it's not clear for me from svg1.1 whether it;s supported or not. It is form image, but for a symbol... I'm not sure.

aside from that, I checked the results on `w3schools.com` and they support this property.
and the main reason why I added it - it's present in some of godot's svgs.

Reply:
I'm a little worried, but for godot !! :)